packages/now-cli/package.json

```diff
@@ -68,7 +68,8 @@
     "@vercel/node": "1.7.3-canary.0",
     "@vercel/python": "1.2.2",
     "@vercel/ruby": "1.2.3",
-    "@vercel/static-build": "0.17.6-canary.1"
+    "@vercel/static-build": "0.17.6-canary.1",
+    "update-notifier": "4.1.0"
```

Comment:
`update-notifier` [is not compatible with `ncc`](https://github.com/vercel/ncc/issues/495), so it's a regular dependency rather than a devDependency.

(Also note that the `update-notifier-webpack` fork that is listed in that GH issue [is still not compatible with ncc](https://github.com/vercel/ncc/issues/495#issuecomment-661311495))

Reply:
We might see `npm install` time go up since [`update-notifier`](https://packagephobia.com/result?p=update-notifier) adds 1 MB and 600+ files.